zh_cn/community/weblogs/index.md

```diff
@@ -4,4 +4,56 @@ title: "博客 - Ruby 官方网站"
 lang: zh_cn
 ---
 
-{% include out-of-date.html %}
+Ruby 相关的博客经过去一年的爆发和相关职位充分发展，
+你能发掘出数以百计的博客，他们共享 Ruby 代码，介绍新技术，
+或者预测 Ruby 的未来。
```

Comment:
![qq 20150611154536](https://cloud.githubusercontent.com/assets/245165/8102244/21b014b8-1051-11e5-8ef4-6ae8e7377014.png)

I refer to the English original, but it is not necessary to add some new lines in chinese.